React/Folder Structure.md

```diff
@@ -0,0 +1,46 @@
+## Folder structure
```

Comment:
Hmm, this should be part of the project structure chapter since we already have a lot of this in there